src/libexpr-test-support/include/nix/expr/tests/meson.build

```
@@ -2,4 +2,8 @@
 
 include_dirs = [ include_directories('../../..') ]
 
-headers = files('libexpr.hh', 'nix_api_expr.hh', 'value/context.hh')
+headers = files(
+  'libexpr.hh', #
+  'nix_api_expr.hh',
+  'value/context.hh',
+)
```

```suggestion
headers = files(
  'libexpr.hh',
  'nix_api_expr.hh',
  'value/context.hh',
  # hack for trailing newline
)
```
Does this work? I would ideally like to keep all the list item lines formatted exactly the same, so having `#....` on its own line.

---

OK great. Does my point of keeping all the list element lines homogenous make sense?

---

Oh, that is an orthogonal thing. I just added the noise so other readers knew why the comment was there, and not remove it by mistake.